app/views/layouts/_edit_store.html.erb

```diff
@@ -2,6 +2,7 @@
     <div class="form-group">
```

Comment:
I would be happy to do that at some point. But i would like to get this branch out and do that on another one if thats cool with you.